recipes/libx265/all/conanfile.py

```diff
@@ -49,7 +49,8 @@ def config_options(self):
             del self.options.with_numa
         # FIXME: Disable assembly by default if host is arm and compiler apple-clang for the moment.
         # Indeed, apple-clang is not able to understand some asm instructions of libx265
-        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch:
+        # FIXME: Disable assembly by default if host is Android for the moment. It fails to build
+        if (self.settings.compiler == "apple-clang" and "arm" in self.settings.arch) or self.settings.os == "Android":
             self.options.assembly = False
```

Comment:
Is this even working? I've submitted https://github.com/conan-io/conan/pull/16672 to show that changing values in config_options will have no effect if the user is setting the option in the CLI

Reply:
If you change the option value in `config_option`, it will change the default value only. In case the user pass `assembly=True`, the `validate_build` will take care and raise and error. Indeed the user has priority over the value configured via `config_options`.

Reply:
This is how it has always worked since Conan 1.x - setting the value of an option in `config_options` makes it the "default" value - and the default value is only used if the user did not override it elsewhere (e.g via CLI).